pkgs/development/compilers/spirv-llvm-translator/default.nix

```diff
@@ -55,10 +55,10 @@ stdenv.mkDerivation {
     })
   ];
 
-  nativeBuildInputs = [ pkg-config cmake spirv-tools ]
+  nativeBuildInputs = [ pkg-config cmake ]
     ++ (if isROCm then [ llvm ] else [ llvm.dev ]);
 
-  buildInputs = [ spirv-headers ]
+  buildInputs = [ spirv-headers spirv-tools ]
```

Comment:
Add it to nativeCheckInputs too

Reply:
I don't think that'll actually do anything, because (ugh, CMake) it just uses a single pkg-config invocation to find spirv-tools.  (If we hacked around that to remove it from the pkg-config search path, we could possibly get CMake to find the two different CMake modules in different places, but as I said in the PR body, I don't think there'd be any actual gain from that.)

Reply:
It won't work if strictDeps is enabled on non-cross but that can be fixed once it's enabled